src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -646,6 +647,9 @@ export default class ExtraproRegistration extends Mixins(DateMixin) {
   @Watch('formValid')
   @Emit('valid')
   private onComponentValid (): boolean {
+    // if we're here it's because the user has changed something
+    this.setHaveChanges(true)
```

Comment:
Normally, this action is attached to various setters in the store.

However, in this component we change properties inside a referenced object, so `setExistingBusinessInfo()` isn't called every time the user changes something. So we do it here. Ditto for ManualBusinessInfo.vue, below.